<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLidarToolsBase</class>
 <widget class="QDockWidget" name="QgsLidarToolsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lidar Tools</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="mToolBoxLidar">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mPageInfo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>282</width>
         <height>306</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Infromation</string>
       </attribute>
       <widget class="QPushButton" name="mButtonChangeGrid">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of points</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mEditNumOfPoints">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>North</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditNorth">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>West</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditWest">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>East</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditEast">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>South</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditSouth">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Z max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditZMax">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Z min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditZMin">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Point density</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditPointDensity">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>211</y>
          <width>281</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Grid index cell size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mEditGridCellSize">
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="mPageDisplay">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>282</width>
         <height>306</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Display controls</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="mButtonElevDisplay">
             <property name="text">
              <string>Create Elevation Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabelElevFile">
             <property name="text">
              <string>_elev.tif</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="mButtonColorDisplay">
             <property name="text">
              <string>Create Color Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabelColorFile">
             <property name="text">
              <string>_rgb.tif</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QPushButton" name="mButtonClassificationDisplay">
             <property name="text">
              <string>Create Classification Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabelClassificationFile">
             <property name="text">
              <string>_class.tif</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QPushButton" name="mButtonIntensityDisplay">
             <property name="text">
              <string>Create Intensity Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabelIntensityFile">
             <property name="text">
              <string>_ints.tif</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QPushButton" name="mButtonHillshadeDisplay">
             <property name="text">
              <string>Create Hillshade Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabelHillshadeFile">
             <property name="text">
              <string>_hill.tif</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
